xwayland/xwm.c

```diff
@@ -467,20 +467,39 @@ static void read_surface_title(struct wlr_xwm *xwm,
 	wlr_signal_emit_safe(&xsurface->events.set_title, xsurface);
 }
 
+static bool check_transient_loop(struct wlr_xwayland_surface *candidate,
```

Comment:
This could be renamed `has_parent`

Reply:
I don't think `has_parent` fits here, maybe `is_parent`?

Reply:
`has_parent` would be fine if the args were reversed. BTW maybe we can find better names for these args.

Reply:
I'll go with `parent` and `child` and `has_parent`. Also renamed the `candidate` variable to `found_parent`.